components/widgets/modals/full_screen_modal.tsx

```diff
@@ -21,20 +21,21 @@ type Props = {
     ariaLabel?: string;
     ariaLabelledBy?: string;
     intl: any; // TODO This needs to be replaced with IntlShape once react-intl is upgraded
+    overrideTargetEvent?: boolean;
 };
 
 class FullScreenModal extends React.PureComponent<Props> {
     private modal = React.createRef<HTMLDivElement>();
 
     public componentDidMount() {
         document.addEventListener('keydown', this.handleKeypress);
-        document.addEventListener('focus', this.enforceFocus, true);
+        document.addEventListener('focus', this.enforceFocus, this.props.overrideTargetEvent);
```

Comment:
From [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener): if you pass a boolean and it's true, any focus event will be override by this listener. On our situation I had to disable it because it was impossible to focus on our textarea.